app/main/innsending/pdf/PdfGenClient.kt

```diff
@@ -36,4 +32,28 @@ class PdfGenClient {
             mapper = { body, _ -> body.readAllBytes() }
         )) { "Response from pdfgen was null" }
     }
+
+    fun ettersendelseTilPdf(innsending: InnsendingNy): ByteArray {
+        val navn = SøkerPdfGen(hentNavn(innsending.personident))
+        val data = SøknadPdfGen(navn, innsending.kvitteringToMap())
+
+        val httpPostRequest = PostRequest(
+            body = data,
+            additionalHeaders = listOf(Header("accept", "application/pdf"))
+        )
+
+        return requireNotNull(
+            httpClient.post(
+                uri = URI.create(ProdConfig.config.pdfGenHost + "/api/v1/genpdf/aap-pdfgen/ettersending"),
+                request = httpPostRequest,
+                mapper = { body, _ -> body.readAllBytes() }
+            )) { "Response from pdfgen was null" }
+    }
+
+    private fun hentNavn(personident: String) = oppslagClientNy.hentNavn(personident).let { navn ->
```

Comment:
Tror du kan bruke destructuring her, husker ikke syntaksen. Noe sånt?
```suggestion
    private fun hentNavn(personident: String) = oppslagClientNy.hentNavn(personident).let { fornavn, etternavn ->
```